docs/pages/components/select.md

```diff
@@ -31,7 +31,7 @@ For lists that require more than 12 options, the <a href="/patterns/combobox-inp
             class="fd-select__control"
             role="button"
             tabindex="0">
-             Select
+             <span class="fd-select__text-content">Select</span>
```

Comment:
Sorry I forgot to say this earlier - can you add these changes to the select storybook examples?